src/game.cpp

```diff
@@ -9942,6 +9944,13 @@ void game::on_options_changed()
 #endif
 }
 
+void game::zero_m_flag_usage_stats()
+{
+    for( int i = 0; i < m_flag::MF_MAX; i++ ) {
```

Comment:
```suggestion
    m_flag_usage_stats.clear();
```
You current code just writes all 0 to it the map entries, but calling `clear` will restore the initial state right after starting the game. That state (the map being empty) will work just fine.